controls/datagrid/overview.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Overview
-page_title: .NET MAUI DataGrid Documentation - Overview
-description: Try out the best-in-class .NET MAUI DataGrid control with out-of-the-box support for operations like sorting, filtering and grouping, editing, and more.
+page_title: .NET MAUI DataGrid Overview
+description: ".NET MAUI DataGrid is a control that allows you to visualize and edit tabular data in your .NET MAUI apps."
```

Comment:
Is using the quotes in the description a new requirement from the SEO team? This was considered obsolete and counterproductive a few years ago.